src/core/Akka.MultiNodeTestRunner.Shared.Tests/MultiNodeTestRunnerDiscovery/DiscoverySpec.cs

```diff
@@ -46,7 +46,11 @@ public void Public_props_and_fields_are_considered()
 
         private static Dictionary<string, List<NodeTest>> DiscoverSpecs()
         {
+#if CORECLR
+            using (var controller = new XunitFrontController(AppDomainSupport.IfAvailable, new System.Uri(typeof(DiscoveryCases).GetTypeInfo().Assembly.CodeBase).LocalPath))
```

Comment:
No, because Assembly.GetExecutingAssembly() isn't available in core 1.1, it's supposed to be back in 2.0, but we have to wait.